recipe/meta.yaml

```diff
@@ -151,7 +147,6 @@ test:
     - pip check
     - python test/smoke_test/smoke_test.py
     - pytest -v {{ ignore_modules }} test/integration_tests
-    # dataset provides the dataset for the hubert example test. It's not a runtime requirement.
```

Comment:
sorry to clarify: are these hubert tests being run? if so, you need `pip install dataset`. If not, the pythonpath setting/exporting lines aren't needed either.

Reply:
they are being run when `run_all_unittests` is set to True. I can wrap them into an if statement.

Reply:
in that case, I think you need to `pip install dataset` somewhere. nm, something for next time. I'm updating this for v2.3 soon so can do it then